src/transformer/utils/get_multichoice_definition.php

```diff
@@ -32,15 +32,25 @@ function get_multichoice_definition(array $config, \stdClass $questionattempt, \
                 ]
             ];
         }, $answers);
+
+        $correctresponsepattern;
+        switch ($interactiontype) {
+            case 'sequencing':
+                $selections = explode('} {', rtrim(ltrim($questionattempt->rightanswer, '{'), '}'));
+                $correctresponsepattern = implode ('[,]', $selections);
+                break;
+            default:
+                $correctresponsepattern = utils\get_string_html_removed($questionattempt->rightanswer);
```

Comment:
Do choices also need to be imploded in the same way?

Reply:
I'm not sure, but I don't think so because I think the multi-choice question is only one option. 

That would be out of scope for this PR anyway. The goal here is to preserve the behavior of the multi-choice question while changing the gap select one.

Reply:
I think it's ok to do this for multichoice too because this is [also an issue for multichoiceset questions](https://github.com/xAPI-vle/moodle-logstore_xapi/blob/master/tests/mod_quiz/attempt_submitted/multichoiceset/statements.json).

Reply:
Ok, since you insist, I fixed for multichoice questions where there are multiple correct answers too. :-)

Waiting for tests to run....

Reply:
Tests passed

Reply:
Haha okay thanks Andrew 👍 looks good